examples/Components/linearsolver/MatrixContributions121.scn

```diff
@@ -18,7 +18,9 @@
     <RequiredPlugin name="Sofa.Component.Visual"/> <!-- Needed to use components [VisualStyle] -->
     <RequiredPlugin name="Sofa.GL.Component.Rendering3D"/> <!-- Needed to use components [OglModel] -->
     <RequiredPlugin name="SofaNewmat"/> <!-- Needed to use components [LULinearSolver] -->
+    <RequiredPlugin name="Sofa.Component.SceneUtility"/> <!-- Needed to use components [InfoComponent] --> 
```

Comment:
```suggestion
```

Reply:
IDK why but I moved this example to the SofaNewMat plugin...